### ISSUE TITLE
Can´t login with Spotify

### ISSUE THREAD

**Issue by nahu2107** [sekaru/spotify-lifx-visualiser#4]
I get the following message "Missing required parameter: client_id"

**Comment by tudddorrr** [comment]
Did you create the .env file?

**Comment by nahu2107** [comment]
> Did you create the .env file?

Yep, the message happens When I´m in localhost:8888 and after I click on the green button "Login with Spotify"

**Comment by tudddorrr** [comment]
Did you fill in all the details in the env file with your own Spotify app details? I.e. your own client id

**Comment by MrRobotjs** [comment]
Same exact problem here. Seems like the app isn't adding the CLIENT_ID and REDIRECT_URI from the .env file into the url after the "Login with Spotify" is clicked on. Even when I manually add the parameters it will just return to "http://localhost:8888" then I get a warning on console about

(node:26112) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

Doesn't seem it would relate to that (maybe?) but yea.

**Comment by MrRobotjs** [comment]
![screenshot](https://i.imgur.com/1H4VCAF.png)

**Comment by tudddorrr** [comment]
Can you show me your .env file and what version of node are you using?
